index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>portfolio mobile</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap" rel="stylesheet"> 
+    <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/ee00ee8a15.js" crossorigin="anonymous"></script>
+</head>
+<body>
+    <div class="header">
+        <nav class="name">
+            <h2>John Doe</h2>
+            <img src="img/Union.png" alt="">
+        </nav>
+    </div>
+    <div class="information">
+        <h1 class="discrptive">Hey There. I'm Alex</h1>
+        <h2 class="discrptive-info">I’am a Software Developer</h2>
+        <p class="paragraph">I can help you build a product , feature or website Look through some of
+                     my work and experience! If you like what you see and have a project you need coded, 
+                    don’t hestiate to contact me.</p>
+        <div class="icon">
+            <a href="#"><i class="fa-brands fa-github"></i></a>
+            <a href="#"><i class="fa-brands fa-linkedin"></i></a>
+            <a href="#"><i class="fa-brands fa-angellist"></i></a>
+            <a href="#"><i class="fa-brands fa-twitter"></i></a>
+            <a href="#"><i class="fa-solid fa-m"></i></a>
+        </div>
```

Comment:
- [ ] The list of icons should use a `ul` tag to group them, and `li` per icon; each icon has a link (a tag):
```
<ul>
   <li><a href="#"><i class="fa-brands fa-github"></i></a></li>
</ul>
```

- [ ] Kindly change the color of icons from white to `#DBD8D7` as provided in Figma design